### PR TITLE
hotdog water/beer

### PR DESCRIPTION
**Pull request by mathis650**
## About the PR 
New drink, hotdog water.

## Why's this needed? 
It was asked a few times and I liked the idea.

## Changelog
`(+)Adds hotdog water, a cool new drink, you get it from distilling hotdogs in a still. Causes minor TOX damage, capped to 10 TOX. Also makes you drunk.
`

### CODE REVIEW

File: code/modules/chemistry/Chemistry-Recipes.dm [hunk 1]
```diff
@@ -4090,3 +4090,11 @@ datum
 			mix_phrase = "The mixture comes together slowly. It doesn't seem like it wants to be here."
 			required_reagents = list("poor_cement" = 1, "silicon_dioxide" = 5, "water" = 1)
 			result_amount = 7
+		hotdogwater
+			name = "Hotdog Water"
```
Comment (BatElite) [inline code/modules/chemistry/Chemistry-Recipes.dm:4094]: For most things it's preferred to put `name` in lowercase. Something about BYOND thinking it's a proper noun or something if I remember.

**Comment by BatElite** [comment]
If only we could actually boil hot dogs.

File: code/modules/chemistry/Chemistry-Recipes.dm (hunk 1 above)
Comment (mathis650) [reply code/modules/chemistry/Chemistry-Recipes.dm:4094]: right, ill see if i can change that

**Comment by mathis650** [comment]
fixed the name
still shows up as water in the reagent scanner, though. color is fine and so are the effects.

File: code/modules/chemistry/Chemistry-Recipes.dm (hunk 1 above)
Comment (mathis650) [reply code/modules/chemistry/Chemistry-Recipes.dm]: fixed it

**Comment by ZeWaka** [comment]
Follow the changelog format and guidelines please

**Pull request closed without merging**